apm-sniffer/apm-sdk-plugin/baidu-brpc-3.x-plugin/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-sdk-plugin</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>8.13.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+
+    <artifactId>baidu-brpc-3.x-plugin</artifactId>
+    <name>baidu-brpc-plugin</name>
+
+    <properties>
+        <brpc-java.version>3.0.5</brpc-java.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.baidu</groupId>
+            <artifactId>brpc-java</artifactId>
+            <version>${brpc-java.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Also, brpc should have been donated to the ASF incubator, what is actually this project about? And is Baidu team still maintaining this?

From here, https://mvnrepository.com/artifact/com.baidu/brpc-java, 3.x was released from 2020-now(2022)

Reply:
Discussion happens here,
https://lists.apache.org/thread/no28p309qpmq29bv0fl7c0m30q9thyjb

We could accept this plugin, as the bprc-java should be renamed soon.

But you need to follow other comments for testing.